### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fluffybird2323**
earlier:  :<
Now:     :>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ You can find the setup instructions [here](SETUP.md)
 
 ## Status of submission
 
-This game was not able to be completed in the time duration for [Jim Jam #2](https://itch.io/jam/jimjam-2) due to some personal engagement I had to attend to at in that time period.
+:<
```
Comment (twaritwaikar) [inline README.md:17]: Can you explain what is the purpose of these?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ You can find the setup instructions [here](SETUP.md)
...
+This game was not able to be completed in the time duration for [Jim Jam #2](https://itch.io/jam/jimjam-2) due to some personal affairs I had to attend to during that time period.
 
-However I will be pushing this game to completion because I like its idea a lot.
+However I will  push this game to completion since I liked its idea a lot. 
+:>
```
Comment (twaritwaikar) [inline README.md:21]: And this too?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Music Guy
 
-Music Guy is a action/rhythm game based on mimicing the beat makeup of a song which in turn help the Music Guy cross a series of cleverly placed obstacles either by jumping over them or by using special abilities.
+Music Guy is a action/rhythm game based on mimicing the beat makeup of a song which in turn helps the Music Guy cross a series of cleverly placed obstacles either by jumping over them or by using special abilities.
```
Comment (twaritwaikar) [inline README.md:3]: Thanks for this 👍

**Review by twaritwaikar — CHANGES_REQUESTED** [review]

**Comment by fluffybird2323** [comment]
Hacktoberfest!

**Comment by twaritwaikar** [comment]
You still need to do something reasonable to benefit any open source repository.

**Pull request closed without merging**